src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -181,23 +185,48 @@ private EpisodeDto MapEpisodeToEpisodeDto(BssEpisode episode)
 
     private async Task ProcessParticipantDataAsync(IEnumerable<Participant> participants, string participantUrl)
     {
+
+        DateTime processingStart = DateTime.UtcNow;
+
+        int successCount = 0;
+        int failureCount = 0;
+        int lastProcessedRow = 0;
+        int rowIndex = 0;
+
         try
         {
-            _logger.LogInformation("Processing participant data.");
+            _logger.LogInformation("Processing started for file at {processingStart}", processingStart);
```

Comment:
Should also include the fileName in this log statement

Reply:
fileName will be logged.